code/modules/client/preferences/screen.dm

```diff
@@ -1,11 +1,3 @@
-/datum/preference/toggle/widescreen
-	category = PREFERENCE_CATEGORY_GAME_PREFERENCES
-	savefile_key = "widescreenpref"
-	savefile_identifier = PREFERENCE_PLAYER
-
-/datum/preference/toggle/widescreen/apply_to_client(client/client, value)
-	client.view_size?.setDefault(getScreenSize(value))
```

Comment:
I don't think the pref existing is inherently bad, just because of folks with non widescreen monitors, however many that is.
I don't think we should be designing around it to any particular degree tho, and I don't think any part of it should be config